modules/Virtualizer_module/CMakeLists.txt

```diff
@@ -44,8 +38,7 @@ yarp_add_idl(${EXE_TARGET_NAME}_THRIFT_SRC ${${EXE_TARGET_NAME}_THRIFT_HDR})
 
 # add include directories to the build.
 include_directories(
-  ${CMAKE_CURRENT_SOURCE_DIR}/include
-  ${CMAKE_CURRENT_SOURCE_DIR}/CybSDK/include)
+  ${CMAKE_CURRENT_SOURCE_DIR}/include )
 
 # add an executable to the project using the specified source files.
 add_executable(${EXE_TARGET_NAME} ${${EXE_TARGET_NAME}_SRC} ${${EXE_TARGET_NAME}_HDR}

```

Comment:
Probably a 
```cmake
target_link_libraries(${EXE_TARGET_NAME} PUBLIC CybSDK)
```
is missing.

Reply:
There is already!

https://github.com/kouroshD/walking-teleoperation/blob/d4cc72ad8bca495c84e1010d00d9ec5a607739e1/modules/Virtualizer_module/CMakeLists.txt#L47-L50